File: AccountingBackend/AccountingBackend/Data/JournalMaster.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace AccountingBackend.Data
{
    public class JournalMaster
    {
        [Key]
        public int JournalMasterId { get; set; }
        public string SerialNo { get; set; }
        public string VoucherNo { get; set; }
        public string InvoiceNo { get; set; }
        public DateTime Date { get; set; }
        public decimal Amount { get; set; }
        public string Narration { get; set; }
        public string UserId { get; set; }
        public int VoucherTypeId { get; set; }
        public int FinancialYearId { get; set; }
        public int CompanyId { get; set; }
        public string Status { get; set; }
        public DateTime? AddedDate { get; set; }
        public DateTime? ModifyDate { get; set; }
        [NotMapped]
        public List<JournalDetails> listOrder { get; set; } = new List<JournalDetails>();
        [NotMapped]
        public List<DeleteItem> listDelete { get; set; } = new List<DeleteItem>();
    }
}

File: AccountingFrontendUpdatedVersion/src/app/models/journal-master.ts
import { JournalDetails } from "./journal-details";

export class JournalMaster {
 JournalMasterId : number=0;
 SerialNo : string='';
 VoucherNo : string='';
 InvoiceNo : string='';
 Date : any;
 Amount : number=0;
 Narration : string='';
 UserId : string='';
 VoucherTypeId : number=0;
 FinancialYearId : number=0;
 CompanyId : number=0;
 Status : string='';
 listOrder : JournalDetails[]=[];
 listDelete : any[]=[];
}
